src/auto/enums.rs

```diff
@@ -40,3 +40,181 @@ impl FromGlib<ffi::PangoAlignment> for Alignment {
     }
 }
 
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash)]
+pub enum Stretch {
+    UltraCondensed,
+    ExtraCondensed,
+    Condensed,
+    SemiCondensed,
+    Normal,
+    SemiExpanded,
+    Expanded,
+    ExtraExpanded,
+    UltraExpanded,
+    #[doc(hidden)]
+    __Nonexhaustive(()),
+}
+
+#[doc(hidden)]
+impl ToGlib for Stretch {
```

Comment:
Sorry, I looked to wrong commit, you already remove reexport,
but, please, remove others too.